op-monitorism/faultproof_withdrawals/runbooks/RUNBOOK.md

```diff
@@ -0,0 +1,217 @@
+# Runbook: Incident Response for Faultproof Withdrawals
```

Comment:
A few questions on how this fits into our broader move towards open sourcing runbooks:
1. Are there internal runbooks in notion that this should replace? If so are those notion runbooks updated accordingly?
2. Is this linked to from the notion table we have that tracks all runbooks?
3. Are all alerts updated to point to this runbook to ensure it's discoverable by oncalls?
4. For open sourced runbooks, did we agree to have them live next to the alert code, as opposed to a single repo that tracks all runbooks?

Reply:
1. No this is a new runbook.
2. Not yet, good point, will link it.
3. Alerts are new and points already to this runbook
4. The new agreements is that we share the runbooks via notion first, and in the future in github. This is not mandatory cc: @alfonso-op to confirm.

Reply:
In terms of open-sourcing runbooks, we're going to start with a low-lift trial for now. As @raffaele-oplabs mentioned, starting with open-sourcing a subset of our Notion runbooks, getting feedback from customers / partners and then going from there.

More context [here](https://www.notion.so/oplabs/Operationalizing-New-Protocol-Features-103f153ee16280a1a4f8e8adf8e6a294?pvs=4#118f153ee16280678644c8afecdd7b9f).

Reply:
Got it, great context in there. Thank you both!